### PR TITLE
Add "Latest updates" page

### LINKED ISSUE #1301: [ NEW PAGE ] Latest updates for "get started" section of guides tab
latest updates section will be in the new patterns structure
Screenshot 2024-05-09 at 10 50 18 AM

What should we put here? e.g.

- links to latest release notes for all products?
- link to latest release blog?
- something else?

### PR DESCRIPTION
**Pull request by frankharkins**
Adds a latest updates page with some initial content. Closes #1301.

I've prioritised news that requires user action (i.e. deprecations), but open to other ideas too.

Points from a recent meeting:
- We will want product owners to write a release summary for each release - and specify that the url out to the release notes should specify the version (this is mentioned in the Docs overview and publishing processes page)
- Highlight product blogs, product announcements, and latest versions of each API ref release
- Duplicate "what's new in the docs" from the announcements

### CODE REVIEW

File: docs/start/latest-updates.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## Qiskit 1.3 will drop Python 3.8 support
+
+Qiskit 1.3.0 will be the first release after Python 3.8 reaches its end-of-life
+stage and stops being supported. Qiskit version 1.3.0 and later may use
+features only available in Python 3.9. To use these later versions of Qiskit,
+you'll need to upgrade the version of Python you're using to 3.9 or greater.
+
+See this announcement in the [release notes](/api/qiskit/release-notes/1.1#deprecation-notes)
+
```
Comment (frankharkins) [inline docs/start/latest-updates.mdx:46]: One idea is to add an announcement about the transpiler service when it's out

```markdown
## IBM announces new Qiskit transpiler service

IBM announced its new AI transpiler service at Think this week. Use the
transpiler service to take advantage of IBM Cloud resources and AI-powered
transpiler passes.

Learn how to [use the transpiler service](/transpile/qiskit-transpiler-service).
```

**Comment by frankharkins** [comment]
Also worth noting that we already have https://docs.quantum.ibm.com/announcements; maybe we should link out to that from the sidebar and more actively publish news there?

**Comment by javabster** [comment]
thanks Frank!

I see all these latest updates are hardcoded into the page, can you think of any ways to make this more automated? I like the idea of summarising some of the biggest things that have happened recently with code examples but I think its going to be difficult to maintain. If we cant think of a more automated way to generate this page then I think we should stick with the type of thing you did in the first paragraph - just listing out links of where people can find latest updates.

Some extra things we could link out to:
- product announcements page
- latest Qiskit release blog (this might be something we need to manually update)
- maybe the events page? for news about latest events coming up: https://www.ibm.com/quantum/community#events

Also I think it would be good to split sections out by package - so have a section at the top which is for general updates (e.g. link to product announcements, qiskit blog etc.) then a section of Qiskit SDK (link to release notes and latest release blog), then runtime, then transpiler service, then maybe a community section (with the events link?)

**Comment by frankharkins** [comment]
Closing as we've decided to link out to the announcements page instead.

**Comment by abbycross** [comment]
Reopening!

File: scripts/patterns-reorg/entries.py [hunk 2]
```diff
@@ -84,6 +84,7 @@
         ),
     ),
     Entry("Hello world", slug="/hello-world", from_file="start/hello-world.ipynb"),
+    Entry("Latest updates", slug="/latest-updates", from_file="start/lates-updates.mdx"),
```
Comment (abbycross) [inline scripts/patterns-reorg/entries.py:87]: @Eric-Arellano just double-checking that I added this to the entries.py file correctly?

Comment (Eric-Arellano) [reply scripts/patterns-reorg/entries.py:87]: Other than typo. Thanks for adding this!

```suggestion
    Entry("Latest updates", slug="/latest-updates", from_file="start/latest-updates.mdx"),
```

Comment (abbycross) [reply scripts/patterns-reorg/entries.py:87]: Eek! Thanks!

**Comment by javabster** [comment]
Screenshot 2024-06-21 at 8 58 49 AM

somethings gone wrong with the italics here?

**Comment by javabster** [comment]
other than the italics issue this LGTM!

File: docs/start/latest-updates.mdx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+Keep up with the latest and greatest from Qiskit and IBM Quantum! Gathered here are the the most recent Qiskit package release summaries, documentation updates, blogs, community events, and more.
+
+## Latest release summaries
+
+{/* release summaries written by product owners */}
+
+### Qiskit SDK v1.1.0
+
+*To see the release notes for all versions, visit the [Qiskit SDK release notes].
```
Comment (jyu00) [inline docs/start/latest-updates.mdx:16]: Is this supposed to be a link?

File: docs/start/latest-updates.mdx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+{/* no other release notes available yet */}
+
+## Qiskit IBM Runtime service updates
+
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use. 
+
+### 28 March 2024
+
+Qiskit Runtime primitives now support QASM 3 as the input format for circuits and standard JSON output format. See [Qiskit IBM Runtime REST API](/api/runtime) for examples.
```
Comment (jyu00) [inline docs/start/latest-updates.mdx:80]: I think the official name is `OpenQasm 3`.

File: docs/start/latest-updates.mdx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+### 3 June 2024
+
+Probabilistic error amplification (PEA) error mitigation method is now available as an experimental option for Estimator V2. See the [`EstimatorOptions` API reference](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions) for more details.
+
+### 18 June 2024
+
+The `optimization_level` option is deprecated for Estimator V2 and will be removed no sooner than three months from now.
```
Comment (jyu00) [inline docs/start/latest-updates.mdx:88]: Our deployment is delayed, so we push the removal date back a bit. 

```suggestion
The `optimization_level` option is deprecated for Estimator V2 and will be removed no sooner than 30 September 2024.
```

**Review by jyu00 — APPROVED** [review]
Some comments but otherwise LGTM.

One suggestion though - what do you think about putting a _last updated <date>_ at the top of the page? So people know if something has changed since their last visit.

File: docs/start/latest-updates.mdx (hunk 3 above)
Comment (abbycross) [reply docs/start/latest-updates.mdx]: Fixed!

File: docs/start/latest-updates.mdx (hunk 4 above)
Comment (abbycross) [reply docs/start/latest-updates.mdx]: Actually OpenQASM 3 - good catch

**Pull request merged**